### PR TITLE
remove all occurences of embeds from docs

### PR DESCRIPTION
**Pull request by nikolasburk**

### CODE REVIEW

File: docs/data-modeling.md [hunk 1]
```diff
@@ -87,6 +80,8 @@ On a technical level, a model maps to the underlying structures of the data sour
 - In MongoDB, a model maps to a _collection_
 - In REST, a model maps to a _resource_
 
+> **Note**: There is not data source connector for REST APIs yet. However, this will be enabled soon through an open connector interface.
```
Comment (janpio) [inline docs/data-modeling.md]: *no

Comment (nikolasburk) [reply docs/data-modeling.md]: [I've updated the page once more](https://github.com/prisma/prisma2/pull/444/files). I've removed MongoDB and REST APIs from the list but still include a _note_ that models would map to collections and resources in MongoDB and REST APIs. IMO this helps build a mental model for what a model is, wdyt?

Comment (janpio) [reply docs/data-modeling.md]: Yeah, the PR you linked to is much clearer 👍

Comment (janpio) [reply docs/data-modeling.md]: Yeah, the PR you linked to is much clearer 👍

**Comment by macrozone** [comment]
@nikolasburk is there a reason why this has been removed? I know it was not implemented, but i was expecting that it will be released soon. i think embeds are important, what is the state of that?

**Comment by nikolasburk** [comment]
Hey @macrozone 👋 embedded types are currently not on the [roadmap](https://www.notion.so/prismaio/Prisma-public-roadmap-50766227b779464ab98899accb98295f) unfortunately 😐 

We currently don't have an open feature request for embedded types, if that's something you're looking for, please open a FR request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) 🙂

**Comment by macrozone** [comment]
hi @nikolasburk thank you, i opened e FR https://github.com/prisma/prisma/issues/3014

**Pull request merged**